Update tools to include try-convert (#18245)
dotnet/docs: docs/core/porting/tools.md

  - As a [command-line tool](https://github.com/Microsoft/dotnet-apiport/releases)
  - As a [Visual Studio extension](https://marketplace.visualstudio.com/items?itemName=ConnieYau.NETPortabilityAnalyzer)
- [.NET API analyzer](../../standard/analyzers/api-analyzer.md) - A Roslyn analyzer that discovers potential compatibility risks for C# APIs on different platforms and detects calls to deprecated APIs.

> - [try-convert](https://www.nuget.org/packages/try-convert/) - A .NET Core global tool that can convert a project or entire solution to the .NET SDK, including moving desktop apps to .NET Core. It is not recommended if you have a more complicated build established (e.g., custom tasks/targets/imports), and will reject many project types that are incompatible with .NET Core.

Reviewer: ```suggestion
- [try-convert](https://www.nuget.org/packages/try-convert/) - A .NET Core global tool that can convert a project or entire solution to the .NET SDK, including moving desktop apps to .NET Core. It is not recommended if you have a more complicated build established (such as custom tasks, targets, or imports), and it rejects many project types that are incompatible with .NET Core.
```
